migrations/0039-add-feedback-event-column.ts

```diff
@@ -0,0 +1,55 @@
+import { MigrationInterface, QueryRunner, TableColumn, TableIndex, TableForeignKey } from 'typeorm';
+
+const TABLE_NAME = 'Feedback';
+const OLD_NAME = 'title';
+const NEW_NAME = 'source';
+
+export class AddFeedbackEventColumn1709112961573 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "${TABLE_NAME}" RENAME COLUMN "${OLD_NAME}" TO "${NEW_NAME}"`);
+    await queryRunner.query(`ALTER TABLE "${TABLE_NAME}" ALTER COLUMN "${NEW_NAME}" TYPE text`);
+
+    await queryRunner.addColumn(
+      TABLE_NAME,
+      new TableColumn({
+        name: 'event',
+        type: 'uuid',
+        isNullable: true,
+      }),
+    );
+
+    await queryRunner.createIndex(
+      TABLE_NAME,
+      new TableIndex({
+        name: 'feedback_by_event_index',
+        columnNames: ['event'],
+      }),
+    );
+
+    await queryRunner.createForeignKey(
+      TABLE_NAME,
+      new TableForeignKey({
+        columnNames: ['event'],
+        referencedTableName: 'Events',
+        referencedColumnNames: ['uuid'],
+        onDelete: 'CASCADE',
+      }),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    // Is this correct? I followed example in migration #29
```

Comment:
Yeah, looks ok to me. you can remove this comment

Also I didn't know this but according to ChatGPT apparently creating the index on a column before the foreign key can speed up foreign key creation so good catch